src/cldr/load.ts

```diff
@@ -1,14 +1,26 @@
 // required for Globalize/Cldr to properly resolve locales in the browser.
 import 'cldrjs/dist/cldr/unresolved';
-import { Require } from '@dojo/interfaces/loader';
-import Map from '@dojo/shim/Map';
 import load from '@dojo/core/load';
-import coreRequest, { Response } from '@dojo/core/request';
+import coreRequest from '@dojo/core/request';
+import { Provider } from '@dojo/core/request/interfaces';
 import has from '@dojo/has/has';
+import { Require } from '@dojo/interfaces/loader';
+import Map from '@dojo/shim/Map';
 import Promise from '@dojo/shim/Promise';
 import * as Globalize from 'globalize';
-import supportedMain from './locales';
 import { generateLocales } from '../util/main';
+import supportedMain from './locales';
+
+let requestProvider: Provider;
+
+if (!has('host-browser')) {
+	requestProvider = (<any> require('@dojo/core/request/providers/node')).default;
```

Comment:
should we even be making provider loading  decisions in here? I thought the point of the setProvider stuff was to invert the setting of it to the highest level (normally at the top of an app?)